plugin/trino-google-sheets/src/main/java/io/trino/plugin/google/sheets/SheetsMetadata.java

```diff
@@ -152,4 +159,28 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
     {
         return new ConnectorTableProperties();
     }
+
+    @Override
+    public ConnectorInsertTableHandle beginInsert(ConnectorSession session, ConnectorTableHandle tableHandle, List<ColumnHandle> columns, RetryMode retryMode)
+    {
+        SheetsTableHandle sheetsTableHandle = (SheetsTableHandle) tableHandle;
+        return new SheetsInsertTableHandle(sheetsTableHandle.getTableName(), buildColumnHandles(sheetsTableHandle));
+    }
+
+    @Override
+    public Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session, ConnectorInsertTableHandle insertHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics)
+    {
+        return Optional.empty();
+    }
+
+    private List<SheetsColumnHandle> buildColumnHandles(SheetsTableHandle sheetsTableHandle)
+    {
+        SheetsTable table = sheetsClient.getTable(sheetsTableHandle.getTableName())
+                .orElseThrow(() -> new TableNotFoundException(sheetsTableHandle.toSchemaTableName()));
+
+        AtomicInteger counter = new AtomicInteger(0);
+        return table.getColumnsMetadata().stream()
+                .map(m -> new SheetsColumnHandle(m.getName(), m.getType(), counter.getAndIncrement()))
```

Comment:
Rename m to column. Please avoid abbreviations as much as possible. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-abbreviations